diff --git a/babel.config.js b/babel.config.js
index e754101..9406afc 100644
--- a/babel.config.js
+++ b/babel.config.js
@@ -2,13 +2,13 @@ module.exports = function (api) {
   api.cache(true);
   return {
     presets: [
-      'babel-preset-expo',       // Expo preset for React Native
+      ['babel-preset-expo', { unstable_transformImportMeta: true }], // Enable import.meta polyfill for Hermes
       '@babel/preset-react',     // React preset for JSX
       '@babel/preset-typescript' // TypeScript support
     ],
     plugins: [
       ['@babel/plugin-transform-react-jsx', { runtime: 'automatic' }], // Ensures automatic JSX runtime
-      'react-native-reanimated/plugin' 
+      'react-native-reanimated/plugin'
     ],
   };
 };
\ No newline at end of file
